src/timing.c

```diff
@@ -179,23 +179,27 @@ JL_DLLEXPORT void jl_timing_show_filename(const char *path, jl_timing_block_t *c
 #endif
 }
 
+JL_DLLEXPORT void jl_timing_show_location(jl_sym_t *file, int line, jl_module_t* mod, jl_timing_block_t *cur_block)
+{
+#ifdef USE_TRACY
+    jl_timing_printf(cur_block, "%s:%d in %s",
+                     gnu_basename(jl_symbol_name(file)),
+                     line,
+                     jl_symbol_name(mod->name));
```

Comment:
Maybe it is worth doing something similar as in 

https://github.com/JuliaLang/julia/blob/9dd3090234268cdb942d481b5af720d1efb89a1a/src/timing.c#L161-L171

where the root module is printed as well in case we are in a submodule? Otherwise, it can sometimes be hard to find the package when you only have the name of a submodule with only the file name.